mautic/mautic
# Fix operators for text type report filters

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
We use on  few reports text type for reports columns setup. 
This type does not support startsWith,endsWith, contains return error.

Related error log:

`[2020-11-27 10:28:53] mautic.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedMethodException: "Attempted to call an undefined method named "contains" of class "Doctrine\DBAL\Query\Expression\ExpressionBuilder"." at .../app/bundles/ReportBundle/Builder/MauticReportBuilder.php line 487 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedMethodException(code: 0): Attempted to call an undefined method named \"contains\" of class \"Doctrine\\DBAL\\Query\\Expression\\ExpressionBuilder\". at ...app/bundles/ReportBundle/Builder/MauticReportBuilder.php:487)"} []`





#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create UTM codes report
3. Choose any columns and add to filter 
UTM contains xxx
  
![image](https://user-images.githubusercontent.com/462477/100456669-168b9580-30c1-11eb-9bef-5d8c1eb75a17.png)
4. Save and check report
5. Before apply PR return error 500
